simtools/applications/db_development_tools/add_new_parameter_from_json_file_to_db.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a new parameter to the sites collection in the DB
+    using a json file as input.
+
+    This application should not be used by anyone but expert users and not often.
+    Therefore, no additional documentation about this applications will be given.
```

Comment:
Experts and developers also need documentation (and tests). I thought with the PR approved we wouldn't need an additional application.